README.md

```diff
@@ -29,23 +29,62 @@
+
+We also provide example projects which use the above, including;
+   - **[Basic light client demo](./projects/demo)**: a demo using `@substrate/connect` to obtain information about chains on the Polkadot network.
+   - **[Light client extension demo](./projects/extension/)**: an example of a browser extension which provides a light client.
+   - **[Wallet extension demo](./projects/wallet-template/)**: an example of a browser extension which provides a full Polkadot wallet leveraging a light client.
+
+Finally, we have:
+- A [step by step guide](./projects/wallet-template/STEP-BY-STEP-GUIDE.md) on how to integrate a light client into a browser extension.
+- Details on [the discovery protocol](./packages/discovery/), including how to implement the browser or extension side of it.
+"""
+
+Because:
+- I feel like those are the main exports here (but perhaps there's something else worth a mention right here too?). Basically we provide a light client, a protocol for discovering relevant extensions, and a way to easily embed the light client in other extensions. Then the rest are auxiliary packages and examples/demos of how these work.
+- I think we need a clear entry point into the documentation, so it's easy for somebody to start here and know exactly where to go to do whatever it is they want to do.
+- We should link to STEP_BY_STEP_GUIDE.md somewhere prominent, otherwise it's easily missed, especially hidden away in the wallet-template project. Perhaps it needs to be more prominent than the one line above too. Not sure!
+*/
+
 ### Overcoming Browser Limitations
 
-Browser limitations on websockets from HTTPS pages make establishing a robust number of peers challenging, as many nodes must be available with TLS. 
+Browser limitations on websockets from HTTPS pages make establishing a robust number of peers challenging, as many nodes must be available with TLS.
 Substrate Connect addresses this issue through a powerful browser extension, allowing chains to stay synced in the background, thereby significantly enhancing the performance of your applications.
 
+/*
+This section feels like it could be more of a "why embed a light client in browser extensions?", which would be cool to have here after the intro.
+
+Then the why is something like:
+- Share one light client across multiple dApps, so that it only needs to spend time starting up (syncing) once, and we avoid slowing down dApps etc
+- Overcoming browser limitations, enabling a more robust connection to the Polkadot ecosystem (or something like that)
+*/
+
 ### Seamless Integration
 
 When building an application with Substrate Connect, it automatically detects whether the user has the extension installed and utilizes it. If not, it seamlessly creates the WASM light client in-page for them. Built on [Polkadot JS](https://polkadot.js.org/docs/api), Substrate Connect ensures that your development experience is as smooth as using a traditional RPC server node.
 
+/*
```

Comment:
Not entirely sure about this. The repo's name is still @substrate/connect abeit we are moving away from @substrate/connect being the main focus and _may_ even kill the library all together. perhaps the repo should be re-named?

i agree it can be moved into the @substrate/connect package readme though.